test/templates/commented-styles.beard

```diff
@@ -1,15 +1,15 @@
 <span class="foo">test</span>
```

Comment:
Can you also test an example comment within a style declaration? Like:

```
.foo {
  // color: green
}
```

and 

```
.foo {
  /* color: green */
}
```

and sprinkle more other style declarations that aren't commented out around to make sure the regex doesn't have an unhandled case?